src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -134,7 +134,7 @@ const CreateConceptForm = props => (
         </select>
       </div>
 
-      {props.concept.toString().trim() === 'question' ?
+      {props.concept.toString().trim() === 'question' ? (
```

Comment:
Is this also about testing the untested areas?

Reply:
No, this is a formatting due to eslint, let me remove it.